docs/DeveloperGuide.adoc

```diff
@@ -729,9 +729,58 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |include email service | email my contacts through the app directly
+
+|`* * *` |user |sort the contact list by Name/contact number | make sense of the data
+
+|`* *` |user |see the next closest search results if search fails | find a person without getting their name exactly correct (fuzzy searching)
+
+|`* *` |user |have a quick reference bar pop up whenever a user enters a single word command (e.g. 'add' will show an example of the full add command structure) | use commands without always having to memorise the command structure or always open the help window
+
+|`* *` |user |get the time where the user is added | find person based on time added
+
+|`* *` |user |use shortcuts for commands | save time and access information easily
+
+|`* *` |user |be able to see the most searched person | find most relevant person
+
+|`* *` |user |be able to complete the command words itself | dont need to remember the command names
```

Comment:
Note that the template for a user story is *As a …​ | I want to …​ | So that I can…*

So apply that template here, the full user story here is *As a user, I want to be able to complete the command words itself so that I can don't need to remember the command names* How can you improve this sentence?